src/lib/firebase.js

```diff
@@ -6,14 +6,15 @@ import 'firebase/firestore';
 // Initalize Firebase.
 // These details will need to be replaced with the project specific env vars at the start of each new cohort.
 var firebaseConfig = {
-  apiKey: "AIzaSyCejH0C07KHO9dkIotMDZVBqbmfdHhe4-s",
-  authDomain: "tcl-25-shopping-list.firebaseapp.com",
-  projectId: "tcl-25-shopping-list",
-  storageBucket: "tcl-25-shopping-list.appspot.com",
-  messagingSenderId: "297367476509",
-  appId: "1:297367476509:web:f0edb0eee0a15b24fa69f4"
+  apiKey: 'AIzaSyCejH0C07KHO9dkIotMDZVBqbmfdHhe4-s',
```

Comment:
I wonder if the changes here were just because of optical reasons?

Reply:
It's how my VS Code formatting is set up